src/main/java/Steps/MyStepdefs.java

```diff
@@ -35,31 +37,32 @@ public void setup() {
 
     @Given("^User is created$")
     public void userIsCreated() throws Throwable {
-        Post user1 = new Post(23,"test","test","test","test","test");
-
-
-        GsonBuilder builder = new GsonBuilder();
-        Gson gson = builder.create();
-
+        long currentTimestamp = System.currentTimeMillis();
```

Comment:
If we use test/test/test all the time it might happens that we can't get what data created when.
So i recommend to generate random data (in my case I use system timer - time in milliseconds)